### ISSUE TITLE
the evaluation results are all 0

### ISSUE THREAD

**Issue by LifeiBanana** [SebastianJanampa/DETRPose#5]
Hi, thank you for your great work!

I’m encountering an issue when training with CrowdPose. During evaluation, all the result metrics are zero, as shown below:

```
Accumulating evaluation results...
DONE (t=0.12s).
IoU metric: keypoints
Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets= 20 ] =  0.000
Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets= 20 ] =  0.000
Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets= 20 ] =  0.000
Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 20 ] =  0.000
Average Recall     (AR) @[ IoU=0.50      | area=   all | maxDets= 20 ] =  0.000
Average Recall     (AR) @[ IoU=0.75      | area=   all | maxDets= 20 ] =  0.000
Average Precision  (AP) @[ IoU=0.50:0.95 | type=  easy | maxDets= 20 ] = 0.000
Average Precision  (AP) @[ IoU=0.50:0.95 | type=medium | maxDets= 20 ] = 0.000
Average Precision  (AP) @[ IoU=0.50:0.95 | type=  hard | maxDets= 20 ] = 0.000
```

Do you have any suggestions on how to resolve this or what might be causing the problem? Thank you in advance for your help!

**Comment by SebastianJanampa** [comment]
Hello @LifeiBanana,

I'm sorry for not getting back to you sooner. In case you didn't solve it. This is a normal behavior because of the warm-up (first 2000 iterations). For DETRPose, we use a batch size of 32, and Crowdpose has 10000 images for the training set. This gives us a total of 312 iterations per epoch. So you should see significant improvement starting at epoch 7
